src/backend/src/game/game.service.ts

```diff
@@ -3,25 +3,82 @@ import { InjectRepository } from '@nestjs/typeorm';
 import { Player } from '../entities/player.entity';
 import { Repository } from 'typeorm';
 import { Game } from '../entities/game.entity';
+import { Round } from '../entities/round.entity';
+import { Grab } from 'src/entities/grab.entity';
+
+const MAX_POINTS = 200;
+const MAX_ROUND_COUNT = 10;
+
+type GameRoundID = {
+  gameId: number;
+  roundId: number;
+};
 
 @Injectable()
 export class GameService {
   constructor(
     @InjectRepository(Player)
     private playersRepository: Repository<Player>,
+    @InjectRepository(Game)
+    private gamesRepository: Repository<Game>,
+    @InjectRepository(Round)
+    private roundRepository: Repository<Round>,
   ) {}
 
-  async create(playerIds: number[]): Promise<number> {
+  // create the initial round
+  async create(playerIds: number[]): Promise<GameRoundID> {
     const players = await Promise.all(
       playerIds.map(async (id) => {
         return await this.playersRepository.findOne(id);
       }),
     );
 
-    const game = await Game.create({ rounds: [], ongoing: true, players });
+    const game = await Game.create({
+      ongoing: true,
+      players,
+    }).save();
+
+    const newRound = await Round.create({
+      roundNumber: 1,
+      pointsRemaining: MAX_POINTS,
+      game,
+    }).save();
+
+    return { gameId: game.id, roundId: newRound.id };
+  }
 
-    await game.save();
+  // get points remaining
+  async getSumPoints(roundId: number): Promise<number> {
+    const round = await this.roundRepository.findOne(roundId, {
+      relations: ['playerMoves'],
+    });
+    const prevSumPoints = round.pointsRemaining;
+    const sumPoints = (acc, cur: Grab) => acc + cur.howMany;
+    const totalGrabs = round.playerMoves.reduce(sumPoints, 0);
```

Comment:
Meganitpick, but I don't know if it's worth it to declare this constant function. You might as well just put it in the function